CHANGES.md

```diff
@@ -0,0 +1,79 @@
+# Changes
+
+Here follows a list of the changes between `v7` and `v8` API versions:
```

Comment:
I would use just the version number as it will be the CHANGELOG file. We can use that for future changes as well. It is something I had I my tasks for a while now.